pkgs/racket-doc/json/json.scrbl

```diff
@@ -80,35 +81,53 @@ the @rfc for more information about JSON.
   the range of @tt{U+10000} and above are encoded as two @tt{\uHHHH}
   escapes, see Section 2.5 of the @|rfc|.
 
+  @racket[indent] is used to indicate format style. If it is
+
+  @itemize[
+     @item{@racket[#f], the converted JSON won't be formatted.}
+     @item{@racket[#\tab], it is used as white space for indenting.}
+     @item{a natural number, it indicates the number of whitespaces to
+             use as white space for indenting.}]
+
 @examples[#:eval ev
   (with-output-to-string
-    (λ () (write-json #hasheq((waffle . (1 2 3))))))
+    (λ () (write-json #hasheq([waffle . (1 2 3)]))))
   (with-output-to-string
-    (λ () (write-json #hasheq((와플 . (1 2 3)))
+    (λ () (write-json #hasheq([와플 . (1 2 3)])
                       #:encode 'all)))
+  (for ([indent (in-list '(#f 0 4 #\tab))])
+    (newline)
+    (write-bytes
```

Comment:
Why `write-bytes` + `with-output-to-bytes`? Couldn't we just `write-json` directly?